models/review.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require("mongoose");
+
+const reviewSchema = mongoose.Schema({
+  recipeId: {
+    type: String,
+    required: true,
+  },
+  reviewer: {
```

Comment:
this should be an id, right? rename to `reviewerId`? Or can you reference it to the User table?